internal/state/block/ownership/block.go

```diff
@@ -22,11 +22,11 @@ func NewService(tx storage.Tx) *service {
 	}
 }
 
-func (s *service) SetCurrentEvoBlockForOwnershipContract(contract string, number uint64) error {
+func (s *service) SetCurrentEvoEventIndexForOwnershipContract(contract string, number uint64) error {
```

Comment:
I've changed the prefix name. 

I would not create another package. I would rather name the `block` package into `synchronization` or something like that because its purpose is to deal with that stuff, right? what do you think?

Reply:
Ok. Either "sync" or "synchronization", as you wish

Reply:
changed to sync